templates/inspec/examples/attributes/old_attributes.yml

```diff
@@ -0,0 +1,5 @@
+gcp_project_id: chef-gcp-inspec
```

Comment:
Why old_attributes.yml?

Reply:
The current integration tests existing for inspec-gcp resources work off of a ruby configuration file that exists in the inspec-gcp repo. This allows me to define attributes with the same names within MM for use in defaults and documentation without copying that file into MM